src/engine/BMInterfaceNewuser.php

```diff
@@ -299,6 +298,15 @@ public function reset_password($playerId, $playerKey, $password) {
             $statement->execute(array(':id' => $playerId,
                                       ':password' => $passwordHash));
 
+            // Now delete the verification key so it can't be reused
+            $query =
+                'DELETE FROM player_reset_verification ' .
+                'WHERE player_id = :player_id ' .
```

Comment:
What about clearing ALL verification keys for this player when successfully using one?

Reply:
Okay.